.github/workflows/goreleaser-snapshot-fleet.yaml

```
@@ -38,7 +38,7 @@ jobs:
     # Docker will not be available.
     #
     # Also not run if author is dependabot (it doesn't have access to Github secrets).
-    if: ${{ (github.repository == 'fleetdm/fleet') && (github.actor != 'dependabot[bot]') }}
+    if: ${{ (github.repository == 'fleetdm/fleet') && (github.actor != 'dependabot[bot]') && (github.event_name == 'pull_request' && (github.event.pull_request.head.repo.full_name == github.repository)) }}
```

```suggestion
    if: ${{ (github.repository == 'fleetdm/fleet') && (github.actor != 'dependabot[bot]') && (github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository) }}
```

The way this is built right now would skip the job on all non-PR activities (pushes, and probably workflow dispatches), which isn't what we want.